doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,226 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
+the shared memory and is responsible for the service discovery, i.e. enabling subscribers to find topics offered by 
+publishers. It also keeps track of all applications which have initialized a runtime and are hence able to use
+publishers or subscribers. To view the available command line options call `$ICEORYX_ROOT/build/iox-roudi --help`.
 
-##### 1. Loan and emplace
+## Runtime
 
-Before sending, we have to loan a chunk of memory to emplace our data.
+Each application that wants to use iceoryx has to instantiate its runtime, which essentially enables communication
+with RouDi. Only one runtime object per user process is allowed.
 
+To do so, the following lines of code are required
 ```cpp
-auto result = publisher.loan(sizeof(CounterTopic));
+iox::runtime::PoshRuntime::initRuntime("some_unique_application_name");
 ```
+## Creating service descriptions for topics
 
-Since the data type is not known to the publisher, we have to provide the size in bytes of the payload data we
-intend to send.
+A ``ServiceDescription`` in iceoryx represents the data to be transmitted and is uniquely identified by three string
+identifiers.
 
-If we successfully acquired a chunk, we can construct the data to be send using placement new and publish it.
-Notice that ``sample.get()`` returns a ``void*`` to the memory ``chunk`` were we can then place our data.
-Depending on what we want to send, we may also use ``memcpy`` to copy the data to the ``chunk``.
+1. ``Group`` name
+2. ``Instance`` name
+3. ``Topic`` name
 
-```cpp
-if (!result.has_error())
-{
-    auto& sample = result.value();
-    void* chunk = sample.get();
-    new (chunk) CounterTopic(73);
-    sample.publish();
-}
-else
-{
-    // could not acquire chunk, handle the error
-}
-```
+A triple consisting of such strings is called a ``ServiceDescription``. The service model of iceoryx is derived
+from AUTOSAR and is still used in the API with these names. The so called canonical protocol is implemented in the
+namespace ``capro``.
 
-Here placement new is required, there is no preconstructed object at ``sample.get()``.
+The following table gives an overview of the different terminologies and the current mapping:
 
-##### 2. Functional approach with loaning
+|         | Group   | Instance         | Topic                  |
+|---------|---------|------------------|------------------------|
+| ROS2    | Type    | Namespace::Topic | -                      |
+| AUTOSAR | Service | Instance         | Event                  |
+| DDS     | -       | -                | /Group/Instance/Topic  |
 
-```cpp
-publisher.loan(sizeof(CounterTopic))
-    .and_then([&](iox::popo::Sample<void>& sample) {
-        new (sample.get()) CounterTopic(73);
-        sample.publish();
-    })
-    .or_else([](iox::popo::AllocationError) {
-        /* handle the error */
-    });
-```
+Service and instance are like classes and objects in C++. So you always have a specific instance of a service during
```

Comment:
Service is related to instance like classes are related to objects in C++. Service describes an abstract topic and an instance is one instantiation of that abstraction. Like an object is an instantiated class.
Events are in this context like members of a class.

Example:

```cpp
class MyRadarService {
   public:
      bool hasObstacleDetected;
      float distanceToObstacle;
};

MyRadarService frontLeftRadarInstance;
std::cout << frontLeftRadarInstance.hasObstacleDetected << std::endl;
```

In the iceoryx world we would subscribe to the service ("MyRadarService", "frontLeftRadarInstance", "hasObstacleDetected") and would receive a sample whenever an obstacle were detected. Or we would subscribe to "distanceToObstacle" and would receive a constant stream of data which presents the distance to the obstacle.